tests/scripts/run_profiler_regressions.sh

```diff
@@ -47,6 +47,12 @@ run_async_mode_T3000_test(){
     fi
 }
 
+run_matmul_stagger_test(){
+    if [ "$ARCH_NAME" == "wormhole_b0" ]; then
```

Comment:
Can we do the check in the pytest script with fixtures?

Reply:
Yes per #7586